specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2022-04-01-preview/apimdeployment.json

```diff
@@ -475,6 +475,55 @@
         "x-ms-long-running-operation": true
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/migrateToStv2": {
+      "put": {
```

Comment:
This api needs to be modelled as a POST not a PUT since you are not actually creating anything.

Reply:
Changed